agents/mcp_servers/deidentification/jsl-deid-mcp-server-v1/test_server.py

```
@@ -0,0 +1,209 @@
+def parse_sse_response(response_text: str) -> Dict[str, Any]:
+    """Parse Server-Sent Events (SSE) response from MCP server."""
+    for line in response_text.split("\n"):
+        if line.startswith("data: "):
+            return json.loads(line[6:])  # Remove "data: " prefix
+    raise ValueError("No data line found in SSE response")
+
+
+def initialize_session() -> str:
+    """Initialize MCP session and return session ID."""
+    print("1. Initializing MCP session...")
+    response = requests.post(
```

Ruff (0.14.14)

Probable use of `requests` call without timeout

(S113)